src/Sylius/Component/Grid/Filter/EntityFilter.php

```diff
@@ -29,8 +29,16 @@ public function apply(DataSourceInterface $dataSource, $name, $data, array $opti
         }
 
         $field = isset($options['field']) ? $options['field'] : $name;
+        $fields = isset($options['fields']) ? $options['fields'] : [$field];
```

Comment:
I guess that we should leave both version, because `field` could be a more common use case